package.json

```diff
@@ -6,13 +6,13 @@
   "description": "Use the right package manager",
   "author": "Anthony Fu <anthonyfu117@hotmail.com>",
   "license": "MIT",
-  "homepage": "https://github.com/antfu-collective/ni#readme",
+  "homepage": "https://github.com/antfu/ni#readme",
```

Comment:
Why do we change back on this?

Reply:
Doing sone test about using `--compact`

Reply:
I Will revert this tmr, or feel free to chabge yourself.

I asked Moha to help us with the behaviour:
- release on every commit on main
- release on any pr using `/pgk-pr-new` or some other comment (like vite ecosytem-ci)